aws/resource_aws_kinesis_analytics_application_test.go

```diff
@@ -29,6 +29,11 @@ func TestAccAWSKinesisAnalyticsApplication_basic(t *testing.T) {
 					resource.TestCheckResourceAttr(resName, "code", "testCode\n"),
 				),
 			},
+			{
+				ResourceName:      resName,
+				ImportState:       true,
+				ImportStateVerify: true,
```

Comment:
Can you please add this `TestStep` to the majority of the other acceptance tests so we can ensure we are properly able to import all configurations? Thanks.